lib/models/globals/globals.dart

```diff
@@ -97,6 +98,8 @@ class Globals extends BaseModel {
 
   String? oidcAuthority;
 
+  dynamic helpUrl;
```

Comment:
Why dynamic type ?

Reply:
It is string in production, bit it is bool in qa

Reply:
Strange, maybe we can discuss this matter with someone from the backend